### DISCUSSION TITLE
How to skip entering name when running `migrate dev`

### DISCUSSION THREAD

**Discussion by paymog** [prisma/prisma#15093] [Q&A] [answered]
I have a workflow in CI/CD when I need to run `prisma migrate dev` but don't have a way to enter a name for the newly created migration. Is there a way to either skip entering a name or to ask prisma to automatically pick a name for the new migration?

**Comment by paymog** [comment, marked as answer]
oh, looks like I can pipe in text like so: `echo '' | prisma migrate dev` and then the new migration will have an empty migration name.

**Reply by janpio** [reply]
`migrate dev` also name a `--name` argument that you can use to provide the name: https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-dev

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
